src/encoding_decoding.cpp

```diff
@@ -74,7 +74,15 @@ toml::array py_list_to_toml_array(const py::list &list) {
       arr.push_back(time_value);
     } else {
       std::stringstream ss;
+#ifdef PYPY_VERSION
+      // see 

```

Comment:
think clang-format is complaining about the space in the end.

Reply:
fixed